google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/executor/AppEngineStandardRunTask.java

```diff
@@ -62,7 +67,9 @@ public void execute(ProcessStartListener startListener) {
         .startListener(startListener)
         .build();
 
-    CloudSdkAppEngineDevServer devServer = new CloudSdkAppEngineDevServer(sdk);
+    AppEngineDevServer devServer = useDevappserver2
```

Comment:
Because devappserver2 is what we consider to be experimental. i.e., the default is devappserver1, 2 if the user overrides the option.